Tasks/JenkinsQueueJobV2/jenkinsqueuejobtask.ts

```diff
@@ -92,6 +95,18 @@ export class TaskOptions {
 
         this.NO_CRUMB = 'NO_CRUMB';
         this.crumb = this.NO_CRUMB;
+
+        this.shouldFail = false;
+        this.failureMsg = '';
+    }
+}
+
+export class taskShouldFail {
```

Comment:
Is this used anywhere?

Reply:
Ah...no it's not. I had tried a different way of failing the task and this is a remnant of it.  I missed removing it when I ripped out the rest.